composed_api.rst

```diff
@@ -218,29 +209,25 @@ Example
 
 ::
 
-    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v1/aoi/123?source=grid
+    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v2/aoi/123/?source=grid
 
 ::
 
     {
-        "GRiD API": {
-            "API Version": "v1"
+        "API Version": "v2", 
```

Comment:
I know this isn't new in v2, but I don't recall the reasoning behind returning the API version in the response. It clearly doesn't hurt anything, but seems redundant in that I specified the API version in my call to the API. To keep things simple, I think this can go.